src/main/java/com/jme3/math/FastMath.java

```diff
@@ -695,13 +695,7 @@ public static float tan(float fValue) {
 	 * @return The integer's sign.
 	 */
 	public static int sign(int iValue) {
-		if (iValue > 0) {
-			return 1;
-		}
-		if (iValue < 0) {
-			return -1;
-		}
-		return 0;
+		return Integer.compare(iValue, 0);
```

Comment:
There is no guarantee that this function returns `-1`, `0`, and `1`. Implementations of the compare merely require values less than 0, equal to 0, or greater than 0.

From the docs:
> Returns:
>     the value 0 if x == y; a value less than 0 if x < y; and a value greater than 0 if x > y

We should therefore keep the original implementation here.

Reply:
I changed it to Integer.signum() then

Reply:
Why do you think there's no guarantee? From OpenJDK implementation, it returns all these possible options and does the same comparison when compared to zero.
https://github.com/openjdk/jdk/blob/f8878cb0cc436993ef1222bc13b00b923d91aad1/src/java.base/share/classes/java/lang/Integer.java#L1486